test/test_ops.py

```diff
@@ -1450,5 +1451,117 @@ def test_is_leaf_node(self, dim, p, block_size, inplace):
         assert len(graph_node_names[0]) == 1 + op_obj.n_inputs
 
 
+class TestFocalLoss:
+    def _logit(self, p: Tensor) -> Tensor:
+        return torch.log(p / (1 - p))
+
+    def _generate_tensor_with_range_type(self, shape, range_type, **kwargs):
```

Comment:
Another small nit: given that these two methods are only used from within `_generate_diverse_input_target_pair` can we maybe inline them there?

Reply:
I see, I guess having them inline make it more explicit to the reader that it is only used on `generate_diverse_input_target_pair`. Will do this.